irohad/ametsuchi/impl/postgres_wsv_common.hpp

```diff
@@ -151,24 +163,24 @@ namespace iroha {
 
     static inline shared_model::builder::BuilderResult<
         shared_model::interface::Peer>
-    makePeer(const pqxx::row &row) noexcept {
+    makePeer(const soci::row &row) noexcept {
```

Comment:
It simplifies code in getPeers() in wsv query, but it is not possible to use soci::row everywhere here.